packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -140,15 +141,24 @@ const start = async (zcf, privateArgs) => {
     )}`,
   );
 
-  /** @type {WeakStore<Brand,PoolFacets>} */
-  const secondaryBrandToPool = makeWeakStore('secondaryBrand');
+  /** @type {Store<Brand,PoolFacets>} */
+  const secondaryBrandToPool = makeStore('secondaryBrand');
   const getPool = brand => secondaryBrandToPool.get(brand).pool;
   const getPoolHelper = brand => secondaryBrandToPool.get(brand).helper;
   const initPool = secondaryBrandToPool.init;
   const isSecondary = secondaryBrandToPool.has;
 
   const quoteIssuerKit = makeIssuerKit('Quote', AssetKind.SET);
 
+  const { publication: metricsPublication, subscription: metricsSubscription } =
```

Comment:
please type this so there is API documentation on what comes out of `getMetrics`. 

e.g. define a type up top,
```
/**
 * @typedef {object} MetricsNotification
 * @property {Brand[]} XYK brands
 */
```
then here would be,
```suggestion
  /** @type {SubscriptonKit<MetricsNotification> */
  const { publication: metricsPublication, subscription: metricsSubscription } =
```

Reply:
done